README.es.md

```diff
@@ -0,0 +1,121 @@
+# sway [![](https://api.travis-ci.org/swaywm/sway.svg)](https://travis-ci.org/swaywm/sway) [![Donate with fosspay](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Español](https://github.com/swaywm/sway/blob/master/README.es.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--)
+
+"**S**irCmpwn's **Way**land compositor" es un gestor de composición de ventanas
+compatible con i3 [Wayland](http://wayland.freedesktop.org/) **en processo de
+desarrollo**.
+Lee el ["FAQ"](https://github.com/swaywm/sway/wiki). Únete al canal de
+["IRC"](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway en
+irc.freenode.net).
+
+**Nota**: Sway se encuentra actualmente "congelado"("frozen"), es decir que
+nuevas características no están siendo desarrolladas, hasta que terminemos de
+integrar Sway y wlroots.
+Las correcciones a fallos "Bug fixes" continúan. Para mayor información
+[lee éste articulo](https://drewdevault.com/2017/10/09/Future-of-sway.html).
+Para mantenerte informado del estado de integración de wlroots, [lee éste
+ticket](https://github.com/swaywm/sway/issues/1390).
+
+[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+
+
+Para el soporte de desarrollo de Sway se puede contribuir a [SirCmpwn's
+Patreon](https://patreon.com/sircmpwn) o se puede contribuir mediante el
+formato de recompensas ["bounties"](https://github.com/swaywm/sway/issues/986)
+para características especificas.
+Cualquier persona es bienvenida a contribuir y reclamar una recompensa.
+Adicionalmente se pueden crear recompensas("bounties") para cualquier
```

Comment:
`recompensas("bounties")` would also expect a whitespace before the `(`.

Reply:
It is a  `[` and not a `(`, is a markdown link.

Reply:
In line 29 it's not a markdown link. `Adicionalmente se pueden crear recompensas("bounties") para cualquier`

Reply:
Ah I see the  problem, this PR was created when the content of the README was a bit diff, this commit removed the reference to the bounties f0ddf6d74